graylog.js

```diff
@@ -18,6 +18,7 @@ var graylog = function graylog(config) {
     this.client       = null;
     this.hostname     = config.hostname || require('os').hostname();
     this.facility     = config.facility || 'Node.js';
+    this.deflate      = config.deflate;
```

Comment:
Could we force this to a boolean here? Perhaps `this.deflate = config.delate !== false;` or something? Then the code below doesn't have to do an explicit check against `false` and the default behavior is defined here instead of on-send.